src/styles/variables.css

```
@@ -0,0 +1,8 @@
+:root {
+  --light-star: url("../images/star-active.svg");
+  --star: url("../images/star.svg");
+  --black-color: #060b35;
+  --grey-color-: #616070;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+}
```

```suggestion
:root {
  --black-color: #060b35;
  --grey-color-: #616070;
  --white-color: #fff;
  --blue-color: #00acdc;
}
```

Separate stars from cards. Move variables to that CSS file.